bittensor/synapse.py

```diff
@@ -454,28 +453,6 @@ def to_headers(self) -> dict:
             if field in headers or value is None:
                 continue
 
-            # Adding the tensor shape and data type to the headers if the object is a tensor
```

Comment:
What were these headers used previously? Was it just metadata about the tensors inside the body?

Reply:
Yes, merely shape-dtype information that overloaded http (same 8kb header error we ran into with text-prompting >1200 tokens). This broke pre-training subnet when the list of tensor header objects approaches >100.